quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -379,6 +384,35 @@ function generateEnvFile {
   echo -e "environment file sourced from: $(BLUE "${ENV_FILE}")"
 }
 
+function generateRouterEnvFile {
```

Comment:
What would this file be used for in the future? Once the router is created, I'm not sure why we would need to refer to the env variables specific to a router unless we were going to recreate it but I'm not sure why someone would want to do that.

Reply:
But for ziti login we already have `zitiLogin` and we have the credentials for that in the quickstart `.env` file, if we're now storing it here as well we're duplicating data and if something changes (like their password) and they forget to update it in one of the env files it could create a headache for a new user to troubleshoot.

And really, the only thing needed to run the router is the path to the yaml file for the router, it seems less complicated to just modify `startExpressEdgeRouter` to take a file path parameter or to create a more generic function `startRouter <path/name.yaml>`

Reply:
this file will be on a node outside of controller.  this is for external edge-router.  the login function are different since it does not use the certs when login to the controller.

the password issue is something we will need to work out next. not sure there is a easier way to handle password change (from the initial passwd). and i am not sure about your comment wrt two places.  the zitiLogin uses environment variable ZITI_USER and ZITI_PWD, so if these were changed by user, when i dump the env file, it should save correctly whether it is on the original environment file or the new one i created for router.